Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook

```diff
@@ -68,15 +66,28 @@
             "isHiddenWhenLocked": true,
             "timeContext": {
               "durationMs": 86400000
-            }
+            },
+            "value": "SYSTEMDB"
+          },
+          {
+            "version": "KqlParameterItem/1.0",
+            "name": "Service",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "value": "",
+            "id": "66894963-736c-4182-9eb1-7aa23ed31ba6"
           },
           {
             "id": "0b81c9d3-446c-4223-b424-f8168f2b5050",
             "version": "KqlParameterItem/1.0",
             "name": "TimePeriod",
             "type": 1,
             "isGlobal": true,
-            "isHiddenWhenLocked": true
+            "isHiddenWhenLocked": true,
+            "value": "2022-11-22T08:06:46.597Z - 2022-12-01T08:11:15.182Z"
```

Comment:
Time difference is hardcoded here?

Reply:
same as previous comment. The value is always passed to the template from the parent workbook. having a default is inconsequential.